packages/flat-lists/package.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "@react-native/flat-lists",
+    "version": "0.72.2",
+    "description": "Flat lists for React Native.",
+    "homepage": "https://github.com/facebook/react-native/tree/HEAD/packages/flat-lists",
+    "repository": {
+      "type": "git",
+      "url": "git@github.com:facebook/react-native.git",
+      "directory": "packages/flat-lists"
+    },
+    "scripts": {
+      "test": "jest"
+    },
+    "license": "MIT",
+    "dependencies": {
+      "@react-native/virtualized-lists": "^0.72.2",
+      "invariant": "^2.2.4",
+      "memoize-one": "^5.0.0"
+    },
+    "devDependencies": {
+      "react-test-renderer": "18.2.0"
+    },
+    "peerDependencies": {
+      "react-native": "*",
+      "react-test-renderer": "18.2.0"
+    }
```

Comment:
I think there's definitely something wrong here. So, instead of me wasting precious CI compute, may be anyone can help me figure this out??

Though, Nicola had commented on the same thing earlier, but that was failing ci, as there was no monorepo setup at that time..

Reply:
false positive :)